__tests__/app.test.js

```diff
@@ -128,6 +128,47 @@ describe("GET /api/articles/:article_id", () => {
   });
 });
 
+describe("GET /api/articles/:article_id/comments", () => {
+  test("200: returns an array of comment objects with the correct properties", () => {
+    return request(app)
+      .get("/api/articles/1/comments")
+      .expect(200)
+      .then(({ body }) => {
+        const { comments } = body;
+        expect(comments).toBeInstanceOf(Array);
+        expect(comments).toHaveLength(11);
+        expect(comments[0].created_at).toBe("2020-11-03T21:00:00.000Z");
+        comments.forEach((comment) => {
+          expect(comment).toEqual(
+            expect.objectContaining({
+              author: expect.any(String),
+              body: expect.any(String),
+              comment_id: expect.any(Number),
+              created_at: expect.any(String),
+              votes: expect.any(Number),
+            })
+          );
+        });
+      });
+  });
+  test("404: returns an err if no articles found by specific id", () => {
+    return request(app)
+      .get("/api/articles/100/comments")
+      .expect(404)
+      .then(({ body }) => {
+        expect(body.message).toBe("No articles found with this ID");
+      });
+  });
+  test("400: returns an error if not a valid input type", () => {
+    return request(app)
+      .get(`/api/articles/durian/comments`)
+      .expect(400)
+      .then(({ body }) => {
+        expect(body.message).toBe("Invalid input");
+      });
+  });
+});
+
```

Comment:
We need add another test for what happens if we have a valid article that exists, but has no comments!